### PR TITLE
RA2 - Added Rhino Tank

### PR DESCRIPTION
**Pull request by MustaphaTR**
Added Soviet Rhino Tank. I tried ingame and looks dark, i don't know why.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
The dark coloring is probably the misbehaving normals (which are also present on other voxels), so it isn't your fault.

File: weapons.yaml [hunk 1]
```diff
@@ -564,3 +564,75 @@ DoublePistolsE:
 		ValidImpactTypes: Water
 	Warhead@4Smu: LeaveSmudge
 		SmudgeType: Crater
+
+120mm:
+	ReloadDelay: 65
+	Range: 5c0
```
Comment (Phrohdoh) [inline weapons.yaml]: This should be `5c768`.

File: weapons.yaml [hunk 2]
```diff
@@ -564,3 +564,75 @@ DoublePistolsE:
...
+	Warhead@3EffWater: CreateEffect
+		Explosion: small_watersplash
+		ImpactSound: ssplash3.aud
+		ValidImpactTypes: Water
+	Warhead@4Smu: LeaveSmudge
+		SmudgeType: Crater
+
+120mmE:
+	ReloadDelay: 80
+	Range: 5c0
```
Comment (Phrohdoh) [inline weapons.yaml]: `5c768`

**Comment by Phrohdoh** [comment]
Needs those two ranges adjusted and a rebase, then I can test.

**Comment by abcdefg30** [comment]
Needs a rebase again.

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by abcdefg30** [comment]
Looks good to me.

**Comment by abcdefg30** [comment]
Thanks :)

**Pull request merged**